project/Build.scala

```diff
@@ -118,8 +118,8 @@ object DottyBuild extends Build {
 
       ("-DpartestParentID=" + pid) :: tuning ::: agentOptions ::: travis_build ::: fullpath
     }
-  ) ++ addCommandAlias("partest", ";test:compile;lockPartestFile;test:test;runPartestRunner") ++ 
-       addCommandAlias("partest-only", ";test:compile;lockPartestFile;test:test-only dotc.tests;runPartestRunner")
+  ) ++ addCommandAlias("partest", ";test:package;lockPartestFile;test:test;runPartestRunner") ++
```

Comment:
@DarkDimius you need `compile:package`, `test:package` produces `dotty_2.11-0.1-SNAPSHOT-tests.jar` instead of `dotty_2.11-0.1-SNAPSHOT.jar`.